src/sentry/api/endpoints/organization_index.py

```diff
@@ -222,23 +219,24 @@ def post(self, request: Request) -> Response:
                             team=team, organizationmember=om, is_active=True
                         )
 
-                    org_setup_complete.send_robust(
-                        instance=org, user=request.user, sender=self.__class__
-                    )
+                org_setup_complete.send_robust(
+                    instance=org, user=request.user, sender=self.__class__
+                )
 
-                    self.create_audit_entry(
-                        request=request,
-                        organization=org,
-                        target_object=org.id,
-                        event=audit_log.get_event_id("ORG_ADD"),
-                        data=org.get_audit_log_data(),
-                    )
+                self.create_audit_entry(
+                    request=request,
+                    organization=org,
+                    target_object=org.id,
+                    event=audit_log.get_event_id("ORG_ADD"),
+                    data=org.get_audit_log_data(),
+                )
 
-                    analytics.record(
-                        "organization.created",
-                        org,
-                        actor_id=request.user.id if request.user.is_authenticated else None,
-                    )
+                analytics.record(
+                    "organization.created",
+                    org,
+                    actor_id=request.user.id if request.user.is_authenticated else None,
+                )
+                om.outbox_for_update().drain_shard(max_updates_to_drain=10)
```

Comment:
what is max_updates_to_drain? Seems magic number-y. We have to drain them all, no?

Reply:
I do, want to change the way that drainy works.  I content against draining them all, but I also agree this is magic numbery.  I have some pending work to improve draining and simplify it, but I don't want to mix it here.  This is consistent with existing behaviors.